packages/astro/src/@types/astro.ts

```diff
@@ -855,12 +855,27 @@ export interface AstroAdapter {
 	args?: any;
 }
 
+export interface APIContext {
+	params: Params;
+	request: Request;
+}
+
 export interface EndpointOutput<Output extends Body = Body> {
 	body: Output;
 }
 
+interface APIRoute {
+	(context: APIContext, request: Request): EndpointOutput | Response;
```

Comment:
```suggestion
	(context: APIContext): EndpointOutput | Response;
```

Is there a reason `request` is passed as a second argument here? I assumed `APIContext` covered that...